### DISCUSSION TITLE
Weapon Variant: Seeking

### DISCUSSION THREAD

**Discussion by ntseng** [Imaginary-Horizons-Productions/prophets-of-the-labyrinth-Archive#263] [Ideas]
For random targeting weapons, changes targeting to select the enemy with the lowest hp.

**Comment by doubledh** [comment]
SEEKING FIRE CRACKERZZZ

**Reply by ntseng** [reply]
Sadly, the implementation of this is a bit of a pain.

**Reply by ntseng** [reply]
Additionally, Firecracker already has 3 upgrades...
